<launch>
  <include file="$(find resources)/launch/camera/run_cam_opencv.launch"/>

  <node pkg="camera_calibration" name="cameracalibrator" type="cameracalibrator.py"  output="screen" args="--size 7x5 --square 0.108">
    <remap from="/image" to="/camera/image_raw"/>
    <remap from="/camera" to="/camera"/>
  </node>
</launch>
